draft-ietf-rtcweb-jsep.xml

```diff
@@ -4006,12 +4006,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         session.</t>
 
         <t>The code flow below shows Alice's endpoint initiating the
-        session to Bob's endpoint. The messages from Alice's JS to
-        Bob's JS are assumed to flow over some signaling protocol via a
-        web server. The JS on both Alice's side and Bob's side waits
+        session to Bob's endpoint. The messages from the Javascript
+        application in Alice's browser, abbreviated AliceJS,  to
+        Javascript in Bob's browser, abbreviated BobJS,
+        are assumed to flow over some signaling protocol via a
+        web server. The Javascript on both Alice's side and Bob's side waits
         for all candidates before sending the offer or answer, so the
         offers and answers are complete; trickle ICE is not used. Both
-        Alice and Bob are using the default bundle policy of
+        user agents in Alice and Bob's browsers, abbreviated as AliceUA
```

Comment:
We haven't used "user agent" in JSEP, preferring "JSEP implementation". It might make sense to use "user agent" here, since it abbreviates nicely, but we should make it clear that when we say user agent, we are referring to a JSEP implementation.